text/0060-tasks-api.md

```diff
@@ -176,6 +178,16 @@ The motivation is to stabilize the current `update` resource to a version that c
 | distinctAttribute | `distrinctAttribute` payload string |
 | displayedAttributes | `displayedAttributes` payload array |
 
+##### dumpCreation
+
+| name    | description  |
+| -----   | ------------ |
+| dumpUid | The generated uid of the dump |
+
+Since the creation of a dump is not a task associated with a particular index, it will only be visible on the `GET` - `/tasks` and `GET` - `tasks/:task_uid` endpoints.
```

Comment:
```suggestion
Since the creation of a dump is not a task associated with a particular index, it is only present on the `GET` - `/tasks` and `GET` - `tasks/:task_uid` endpoints.
```